src/utils/types.ts

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const tryGetString = (obj: any): string | null => {
+  if (typeof obj.toString === 'function') return obj.toString()
+  else return null
+}
```

Comment:
I personally find the following version simpler, but again, no strong opinion here:

```js
export const tryGetString = (obj: any): string | null => (
  typeof obj.toString === 'function' ? obj.toString() : null
)
```

Reply:
Yeah, it takes a while to get used to the "no return" statements. Maybe this is simpler then:

```js
export const tryGetString = (obj: any): string | null => {
  return typeof obj.toString === 'function' ? obj.toString() : null
}
```

but anyway, let's not waste more time on this :)

Reply:
It's good to agree on common patterns. Why not go for your last proposition which seems like a good compromise